### DISCUSSION TITLE
How to set up a node as non routing node?

### DISCUSSION THREAD

**Discussion by iamn3mo** [Ride-The-Lightning/RTL#1370] [Q&A] [answered]
Hi all,
I'd like to set up my node as a non routing node. I just want to accept payments and rebalance manually from time to time. So far I achieved this by only having one big channel. However, for obvisious reasons that's not optimal. How would I achieve my goal of a non routing node with multiple open channels?

**Comment by saubyk** [comment, marked as answer]
Hi, you have multiple options:
- You can open private channels exclusively. But keep in mind that invoice to receive over private channels should always have a routing hint (This is supported in RTL. You can click `Private Routing Hints` toggle, every time you generate an invoice)
- You can open public channels, but set an extremely high routing fee. This will ensure that the channels are not picked up for routing

Any of the above option would give you a non-routing node

**Comment by iamn3mo** [comment]
Thank you
